fdbclient/NativeAPI.actor.cpp

```diff
@@ -9890,18 +9890,42 @@ ACTOR Future<bool> setBlobRangeActor(Reference<DatabaseContext> cx, KeyRange ran
 			tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
 			tr->setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
 
+			state Standalone<VectorRef<KeyRangeRef>> blobRanges;
+			state int batchLimit = 10000;
```

Comment:
Not that I think this would ever be a reasonable use case, but to confirm my understanding - this means that unblobbifying 10001+ consecutive ranges would fail even if the boundaries are aligned?